step-release-vis/src/app/services/environment.ts

```diff
@@ -26,44 +33,8 @@ export class EnvironmentService {
     return [];
   }
 
-  // TODO(andreystar): add json from file
```

Comment:
do we have a file for these already in master?

Reply:
The commits were supposed to be separate pr's, fixed that now.